act/plotting/timeseriesdisplay.py

```diff
@@ -937,8 +946,9 @@ def plot_barbs_from_u_v(
             else:
                 units = ''
             ytitle = ''.join(['(', units, ')'])
-            num_y = ydata.shape[0]
-            barb_step_y = round(num_y / num_barbs_y)
+            if barb_step_x is None:
```

Comment:
Should this be if barb_step_y is None instead of x?

Reply:
Good catch, updating!